src/pkey/clu_rsa.c

```diff
@@ -243,6 +243,8 @@ int wolfCLU_RSA(int argc, char** argv)
             }
         }
 
+        der = der - derSz;
```

Comment:
Can we add some checking for derSz here before adjusting `der`? It is a signed long. For readability `der -= derSz` is easier.

Reply:
Both good points.